libs/cardano-ledger-test/src/Test/Cardano/Ledger/Constrained/Conway/Certs.hs

```diff
@@ -0,0 +1,18 @@
+-- | Specs necessary to generate, environment, state, and signal
+-- for the CERTS rule
+module Test.Cardano.Ledger.Constrained.Conway.Certs where
+
+import Cardano.Ledger.Conway (ConwayEra)
+import Cardano.Ledger.Conway.Rules
+import Cardano.Ledger.Conway.TxCert
+import Cardano.Ledger.Crypto (StandardCrypto)
+import Constrained
+import Data.Sequence (Seq)
+
+certsEnvSpec ::
+  Specification fn (CertsEnv (ConwayEra StandardCrypto))
+certsEnvSpec = TrueSpec
```

Comment:
These will get more refined in one of the next tickets then?

Reply:
I assume so

Reply:
@teodanciu indeed! This PR just adds the skeleton and makes sure that everything typechecks.